cfn-resources/teams/cmd/resource/team-user/mocks/README.md

```diff
@@ -4,7 +4,7 @@ These mocks are generated through the `mockgen` command.
 
 ## HOW-TO: Update mocks
 
-1. In order to update these mocks, you need to have `mockgen` **installed** on your machine.
+1. In order to update these mocks, you need to have `mockgen` **installed** on your machine: `go install github.com/golang/mock/mockgen@latest`.
```

Comment:
Something to consider: https://github.com/golang/mock

> Update, June 2023: This repo and tool are no longer maintained. Please see [go.uber.org/mock](https://github.com/uber/mock) for a maintained fork instead.

We may want to update the make file `devtools` command to install mock gen as well.

Reply:
Thank you Andrea, I was aware of (left a comment at the start of this PR) and first wanted to check if there were better ideas. I updated to `go.uber.org/mock` now.

> We may want to update the make file devtools command to install mock gen as well.

I am not sure what you mean here but I might have followed a better approach with `go:generate` as suggested by @gssbzn below, as such this readme will be removed.

Reply:
ok added to the `make tools` - thank you!